src/timeseriesflattener/testing/synth_data_generator/synth_col_generators.py

```diff
@@ -47,91 +46,24 @@ def create_outcome_values(
     return out  # type: ignore
 
 
-def generate_text_data(
-    n_samples: int,
-    sequence: str,
-    tokenizer: Optional[Any] = None,
-    model: Optional[Any] = None,
-) -> list[str]:
-    """Generate text data.
```

Comment:
This was used to generate the synthetic text data (that we are still using) - might want to keep it around? I don't think it's imported or called anywhere so the transformer dependency is not required for the package